apps/a11y-tests/src/tests/__snapshots__/TextField.test.ts.snap

```diff
@@ -27,7 +22,7 @@ Array [
       "text": "Fix any of the following: Element has insufficient color contrast of 2.21 (foreground color: #a6a6a6, background color: #f4f4f4, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1.",
     },
     "partialFingerprints": Object {
-      "fullyQualifiedLogicalName": ".prefix-185 > span",
+      "fullyQualifiedLogicalName": ".prefix-240 > span",
```

Comment:
can you call resetIds after each test?

Reply:
The test operates on top of rendered demo page (via headless Chrome), instead of React's internal DOM. I don't think there's a place for it to call `resetIds`..

Reply:
OH these are captures of a11y error messages O.o I didn't expect us to have so many errors and so didn't realize these were snapshots of errors

Reply:
Yeah, I've noticed these ids are fairly unstable, but don't know what to do about it except updating snapshots now and then, as a11y-tests are carried out post-rendering.

Reply:
Could scrub the `-###` suffix from class names in a snapshot serializer? Or would that break something else?

Reply:
I'm a little hesitant on mutating/scrubbing these IDs because down the road we *might* need these info to reveal (e.g. by displaying a highlight box) offending elements.

Will keep an eye on the friction it would cause in the future. If too much then let's scrub those IDs.